ajax/libs/FormChimp/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "FormChimp",
+  "filename": "jquery.formchimp.min.js",
+  "title": "FormChimp",
```

Comment:
Let's remove `"title": "FormChimp",` this field is not necessary field.
Thank you.